src/renderer/backend/dx9.rs

```diff
@@ -90,7 +91,17 @@ impl RenderContext for D3D9RenderEngine {
 
 impl RenderEngine for D3D9RenderEngine {
     type RenderTarget = IDirect3DSurface9;
-
+    fn reset(&mut self) {
+        tracing::error!("Calling reset stuff");
```

Comment:
I'd rather use `tracing::trace!` for this, as this is not an error.

To keep in line with the general crate style, could you please add a `use tracing::{error, ...}` statement at the top? Same for the instances below.